build_IT.xml

```diff
@@ -0,0 +1,146 @@
+        <arg line='-f "--output_wrapper=@{outputwrapper}"' />
+        <arg line='-f "--compilation_level=@{compilationlevel}"' />
+        <arg line='-f "--warning_level=VERBOSE"' />
+        <arg line='-f "--jscomp_error=accessControls"' />
+        <arg line='-f "--jscomp_error=ambiguousFunctionDecl"' />
+        <arg line='-f "--jscomp_error=checkDebuggerStatement"' />
+        <arg line='-f "--jscomp_error=checkRegExp"' />
+        <arg line='-f "--jscomp_error=checkTypes"' />
+        <arg line='-f "--jscomp_error=checkVars"' />
+        <arg line='-f "--jscomp_error=const"' />
+        <arg line='-f "--jscomp_error=constantProperty"' />
+        <arg line='-f "--jscomp_error=deprecated"' />
+        <arg line='-f "--jscomp_error=duplicate"' />
+        <arg line='-f "--jscomp_error=duplicateMessage"' />
+        <arg line='-f "--jscomp_error=es5Strict"' />
+        <arg line='-f "--jscomp_error=externsValidation"' />
+        <arg line='-f "--jscomp_error=fileoverviewTags"' />
+        <arg line='-f "--jscomp_error=globalThis"' />
+        <arg line='-f "--jscomp_error=internetExplorerChecks"' />
+        <arg line='-f "--jscomp_error=invalidCasts"' />
+        <arg line='-f "--jscomp_error=misplacedTypeAnnotation"' />
+        <arg line='-f "--jscomp_error=missingProperties"' />
+        <arg line='-f "--jscomp_error=nonStandardJsDocs"' />
+        <arg line='-f "--jscomp_error=strictModuleDepCheck"' />
+        <arg line='-f "--jscomp_error=suspiciousCode"' />
+        <arg line='-f "--jscomp_error=typeInvalidation"' />
+        <arg line='-f "--jscomp_error=undefinedNames"' />
+        <arg line='-f "--jscomp_error=undefinedVars"' />
+        <arg line='-f "--jscomp_error=unknownDefines"' />
+        <arg line='-f "--jscomp_error=uselessCode"' />
+        <arg line='-f "--jscomp_error=visibility"' />
+        <extraflags />
+      </exec>
+    </sequential>
+  </macrodef>
+
+  <macrodef name="gjslint">
+    <attribute name="inputfile" />
+    <sequential>
+      <exec executable="python" failonerror="false" logError="true">
+        <env key="PYTHONPATH"
+             value="${closure-linter.dir}:${python-gflags.dir}"/>
+        <arg value="${closure-linter.dir}/closure_linter/gjslint.py" />
+        <arg line='--strict' />
+        <arg line='"@{inputfile}"' />
+      </exec>
+    </sequential>
+  </macrodef>
+
+  <condition property="os.iswindows">
+    <os family="windows" />
+  </condition>
+  <condition property="os.isunix">
+    <os family="unix" />
+  </condition>
+  <target name="nul" if="os.iswindows">
+    <property name="null.device" value="NUL" />
+  </target>
+  <target name="devnull" if="os.isunix">
+    <property name="null.device" value="/dev/null" />
+  </target>
+  <target name="setnulldevice" depends="nul,devnull" />
+
+  <target name="clean" description="deletes all generated files">
+    <delete file="dist/libphonenumber.js" />
+    <delete file="dist/libphonenumber_IT.js" />
+  </target>
+
```

Comment:
This is a stripped-down version of build.xml that won't perform any test-related compilation but will concentrate only on the minified Italian executable.